vuepress/docs/next/docs/consume/entandoapp-cr.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# EntandoApp Custom Resource Definition 
+The EntandoApp CRD is the deployment of a Docker image that hosts the Entando and Java-based web application. Server-side components include the Entando App Engine,  Entando Component Manager, Entando App Builder, and the user-facing application.
```

Comment:
extra space before ECM

what is the user-facing app? "your" using-facing instead of "the"?